tests/contrib/test_drf_nested_routers.py

```diff
@@ -88,8 +88,4 @@ def get_queryset(self):
     assert_schema(
         _generate_nested_routers_schema(RootViewSet, ChildViewSet),
         'tests/contrib/test_drf_nested_routers.yml',
-        reverse_transforms=[
-            lambda x: x.replace('format: uuid', 'pattern: ^[0-9]+$'),
-            lambda x: x.replace('\n        description: A UUID string identifying this root.', '')
-        ]
```

Comment:
I didn't proceed with this, but it would be interesting here to keep this information for UUID's as well, as it would expose more correct information to Open API.

```yml
  /root/{id}/:
    get:
      operationId: root_retrieve
      parameters:
      - in: path
        name: id
        schema:
          type: string
          format: uuid
        description: A UUID string identifying this root.
        required: true

```

Another approach that's adds pattern without removing format and description, but would be good to know if it's spec compliant:


```yml
        schema:
          type: string
          format: uuid
          pattern: [0-9]+
        description: A UUID string identifying this root.
```